### PR TITLE
Tunes Unusual Parent Process for cmd.exe rule to exclude oobe activity

### PR DESCRIPTION
**Pull request by hilt86**
When dllhost.exe is called with the "/Processid:{CA8C87C1-929D-45BA-94DB-EF8E6CB346AD}" argument it is creating an "OOBE Elevated Object Server"  as per https://strontic.github.io/xcyclopedia/library/clsid_ca8c87c1-929d-45ba-94db-ef8e6cb346ad.html

Out of the box experience is part of the Windows autopilot and therefore should be legitimate behaviour.

### CODE REVIEW

File: rules/windows/execution_command_shell_started_by_unusual_process.toml [hunk 1]
```diff
@@ -26,9 +26,31 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "windows" and event.type == "start" and
-  process.name : "cmd.exe" and
-  process.parent.name : ("lsass.exe",
+sequence with maxspan=3m 
+[process where host.os.type == "windows" and event.type == "start" and ( ( process.name: "dllhost.exe" and not process.args: ("/Processid:{CA8C87C1-929D-45BA-94DB-EF8E6CB346AD}") ) or ( process.name: ("lsass.exe",
```
Comment (brokensound77) [inline rules/windows/execution_command_shell_started_by_unusual_process.toml:30]: you don't need the `sequence` to add this exclusion. If you just leave the original logic, you can append it with

```sql
  ... and
  not (process.parent.name : "dllhost.exe" and process.parent.args : "/Processid:{CA8C87C1-929D-45BA-94DB-EF8E6CB346AD}")
```

Comment (hilt86) [reply rules/windows/execution_command_shell_started_by_unusual_process.toml:30]: ah ok - I didn't think the process.parent.args was available...let me try that

Comment (hilt86) [reply rules/windows/execution_command_shell_started_by_unusual_process.toml]: Thanks for your tip - not sure why I assumed process.parent.args was not in available!

**Review by brokensound77 — APPROVED** [review]
LGTM, thanks for the contribution!

**Review by w0rk3r — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**